internal/file/group.go

```diff
@@ -47,33 +47,26 @@ func (fileGroup *Group) GetAllFiles() []string {
 	return append(files, fileGroup.LockFiles...)
 }
 
-func (fileGroup *Group) checkFilePathDependantCases(fileMatch bool, lockFileMatch bool, file string) bool {
+func (fileGroup *Group) checkFilePathDependantCases(manifestFileMatch bool, lockFileMatch bool, fileName string) bool {
```

Comment:
No longer needed with your new solution :star:

Reply:
Would you like me to remove this deprecated function?

Reply:
I removed it, we can revert the commit if you want to keep it.